packages/shell-api/src/change-stream-cursor.ts

```diff
@@ -41,7 +41,7 @@ export default class ChangeStreamCursor extends ShellApiWithMongoClass {
       throw new MongoshRuntimeError('ChangeStreamCursor is closed');
     }
     const result = this._currentIterationResult = new CursorIterationResult();
-    return iterate(result, this, this._batchSize ?? await this._mongo._batchSize());
+    return iterate(result, this, this._batchSize || await this._mongo._displayBatchSize());
```

Comment:
I’m not sure I understand why we treat this differently than the other cursors? Is there a particular reason?

Reply:
The `.batchSize()` method for the ChangeStreamCursor does not have an effect apart from the visual batching. The other cursor batch sizes do affect the `getMore()` calls that the driver's cursor internally makes. Hence I figured to treat it differently here as well as in the help for the `batchSize` method.

Reply:
Right … I think it’s very confusing that the `.batchSize()` methods do entirely different things now? I’d probably just remove the method then.

Reply:
Yeah good point... I'll kick it out. And in light of the discussion and NODE-3411 I'd also remove the default batch size options we introduced completely.